From 1951d7c3535d46edbfe16b8a1b8d2176d96d4235 Mon Sep 17 00:00:00 2001
From: allgdante <allan.garret@gmail.com>
Date: Mon, 23 Mar 2020 16:12:36 +0000
Subject: [PATCH 07/12] utils: adjust tests to match the Alpine layout

Here we need to adjust several utilities path with the ones used by
Alpine
---
 utils/test/test-aa.py | 10 ++++------
 1 file changed, 4 insertions(+), 6 deletions(-)

diff --git a/utils/test/test-aa.py b/utils/test/test-aa.py
index b5221737..9d1d3ecc 100644
--- a/utils/test/test-aa.py
+++ b/utils/test/test-aa.py
@@ -157,15 +157,13 @@ class AaTest_get_interpreter_and_abstraction(AATest):
     tests = [
         ('#!/bin/bash',             ('/bin/bash',           'abstractions/bash')),
         ('#!/bin/dash',             ('/bin/dash',           'abstractions/bash')),
-        ('#!/bin/sh',               ('/bin/sh',             'abstractions/bash')),
-        ('#!  /bin/sh  ',           ('/bin/sh',             'abstractions/bash')),
-        ('#!  /bin/sh  -x ',        ('/bin/sh',             'abstractions/bash')),  # '-x' is not part of the interpreter path
+        ('#!/bin/sh',               ('/bin/sh',             None)),
+        ('#!  /bin/sh  ',           ('/bin/sh',             None)),
+        ('#!  /bin/sh  -x ',        ('/bin/sh',             None)),  # '-x' is not part of the interpreter path
         ('#!/usr/bin/perl',         ('/usr/bin/perl',       'abstractions/perl')),
         ('#!/usr/bin/perl -w',      ('/usr/bin/perl',       'abstractions/perl')),  # '-w' is not part of the interpreter path
-        ('#!/usr/bin/python',       ('/usr/bin/python',     'abstractions/python')),
-        ('#!/usr/bin/python2',      ('/usr/bin/python2',    'abstractions/python')),
-        ('#!/usr/bin/python2.7',    ('/usr/bin/python2.7',  'abstractions/python')),
         ('#!/usr/bin/python3',      ('/usr/bin/python3',    'abstractions/python')),
+        ('#!/usr/bin/python3.8',    ('/usr/bin/python3.8',  'abstractions/python')),
         ('#!/usr/bin/python4',      ('/usr/bin/python4',    None)),  # python abstraction is only applied to py2 and py3
         ('#!/usr/bin/ruby',         ('/usr/bin/ruby',       'abstractions/ruby')),
         ('#!/usr/bin/ruby2.2',      ('/usr/bin/ruby2.2',    'abstractions/ruby')),
-- 
2.25.2

